codegen/src/main/java/com/paypal/selion/elements/HtmlSeLionElement.java

```diff
@@ -53,6 +56,20 @@
     public String toString() {
         return String.format("%s, canHaveParent=%s",super.toString(), canHaveParent);
     }
+    
+    /**
+     * By providing the qualified name of a custom element we can register it to the element array.
+     * Custom elements are inserted before SeLion elements, if you use the same name it will overwrite the existing element.
+     * 
+     * @param element string of the qualified class
+     */
+    public static void registerElement(String element) {
+        List<HtmlSeLionElement> temp = new ArrayList<HtmlSeLionElement>(Arrays.asList(values));
+        
+        temp.add(0, new HtmlSeLionElement(HtmlElementUtils.getPackage(element), HtmlElementUtils.getClass(element), true, false));
+        
+        values = temp.toArray(new HtmlSeLionElement[temp.size()]);
```

Comment:
I believe that the implementation has to be like this. For example in #81 I give an example of a ExtJS SelectList. This element is not compliant with <select> and the SeLion Element doesn't work with it. So I want to create a CustomElement similar to the SelectList if I don't put them in the beginning I could not use a name like ExtJSSelectList as the normal SelectList would be picked up first. So that would limit the users naming convention significantly. 

What about adding a check before elements get add if something with that name already exists? If it does I throw an exception. For the users that want to specifically overwrite, I can add a parameter to disable that check.